docstore/mongodocstore/mongo_test.go

```diff
@@ -163,7 +164,17 @@ func TestConformance(t *testing.T) {
 	drivertest.RunConformanceTests(t, newHarness, codecTester{}, []drivertest.AsTest{verifyAs{}})
 }
 
-func newTestClient(t *testing.T) *mongo.Client {
+func TestConformanceV3(t *testing.T) {
+	client := newTestClient(t, serverURIV3)
```

Comment:
These two functions are identical except for the URI. Why not put the body code into a helper function that takes the URL as an arg or alternatively use a loop in a single function?